glide.lock

```diff
@@ -0,0 +1,31 @@
+hash: d6fc1ffda5336bc94cc564496911ac0f6d812481c65b06e2a7833116cf3d127c
+updated: 2017-10-24T09:32:55.826752182-04:00
+imports:
+- name: github.com/envoyproxy/data-plane-api
```

Comment:
not needed (there's no code there)

Reply:
I just copied this from whatever @junr03 gave me. @junr03 any thoughts on this?